doc/build-osx.md

```diff
@@ -64,6 +64,18 @@ Build Bitcoin Core
 
         make deploy
 
+Disable-wallet mode
+--------------------
+When the intention is to run only a P2P node without a wallet, Bitcoin Core may be compiled in
+disable-wallet mode with:
+
+    ./configure --disable-wallet
+
+In this case there is no dependency on Berkeley DB 4.8.
+
+Mining is also possible in disable-wallet mode, but only using the `getblocktemplate` RPC
+call not `getwork`.
```

Comment:
I doubt that such an option even exists: I get `unknown command: getwork`

Reply:
Should I add a commit to update that line in both the Unix and OSX build instructions to the following?

```
Mining is also possible in disable-wallet mode using the `getblocktemplate` RPC call.
```

Reply:
Right, could make sense to remove it in build-unix.md as well.

Though, shouldn't the configure options be independent of operating system? So maybe a doc shared across all os that describes the configure options could make sense. (no strong opinion on this)